### PR TITLE
fix: port bindings

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

**Comment by dejanb** [comment]
@ctron Switching to `::` works for me here (and I see that Bob is using the same for his services). I'm not sure if that will work for others?

File: infrastructure/src/app/http.rs [hunk 1]
```diff
@@ -186,7 +186,7 @@ mod default {
     use super::*;
 
     pub fn bind_addr() -> String {
-        "::1".to_string()
+        "::".to_string()
```
Comment (ctron) [inline infrastructure/src/app/http.rs:189]: I wouldn't do that, we can override in the configuration. But keeping that to "localhost" instead to "wide open" makes more sense IMHO.

**Review by ctron — CHANGES_REQUESTED** [review]

File: infrastructure/src/app/http.rs (hunk 1 above)
Comment (ctron) [reply infrastructure/src/app/http.rs:189]: Here, this is the setting: https://github.com/trustification/trustification/blob/d652456df89b8790a9fae3049b66f8e48e1f80ff/infrastructure/src/app/http.rs#L133

So just going for `HTTP_SERVER_BIND_ADDR=::` should work.

Comment (dejanb) [reply infrastructure/src/app/http.rs]: Fixed

**Review by ctron — APPROVED** [review]

**Pull request merged**